### PR TITLE
Add another workaround alternative

### LINKED ISSUE #433: Add shortcut workaround for firefox in private mode to help article
Suggestion or Edit for: bitwarden.com/help/article/extension-wont-load-in-private-mode/

This article already has two decent workarounds. However, the easiest BY FAR is to just use the shortcut (CTRL + SHIFT + L by default), as mentioned here: https://github.com/bitwarden/browser/issues/136#issuecomment-780717906

I think this would be really nice to add to the help article for all the new people coming from LastPass (me included). I'll open a PR for this, feel free to completely rewrite my suggested wording there, what matters is that it exists as a workaround in the help article :)

### PR DESCRIPTION
**Pull request by MarcusOtter**
Closes #433

Edit: Feel free to completely rewrite everything I wrote! English isn't my first language so I'd even encourage a full rewrite. See this as more of an initiative PR to add this third workaround 😄

### CODE REVIEW

File: _articles/miscellaneous/extension-wont-load-in-private-mode.md [hunk 1]
```diff
@@ -9,13 +9,16 @@ tags: [firefox, extension, private window, private mode, private browsing]
 
 The Bitwarden browser extension does not completely function in Firefox's private browsing mode. This is a known issue specific only to Firefox. You will see a message indicating so when you try to open the Bitwarden popup window in a private window. We have discussed the problem with Mozilla, however, they seem unable to fix it so that extensions like Bitwarden can function entirely in private mode.
 
-As a workaround, if you are looking to auto-fill a login while in a private window you can use the right-click context menu:
+There are currently 3 workarounds for auto-filling a login while in Firefox's private browsing mode:
 
-1. Navigate to the website's login page while browsing in a private window.
-2. Right click the page. A context menu will appear.
-3. Select **Bitwarden** &rarr; **Auto-fill** and then pick your login.
+(1) Use the right-click context menu:
```
Comment (fschillingeriv) [inline _articles/miscellaneous/extension-wont-load-in-private-mode.md:14]:
```suggestion
1. Use the right-click context menu:
```

File: _articles/miscellaneous/extension-wont-load-in-private-mode.md [hunk 2]
```diff
@@ -9,13 +9,16 @@ tags: [firefox, extension, private window, private mode, private browsing]
...
+  1. Navigate to the website's login page while browsing in a private window.
+  2. Right click the page. A context menu will appear.
+  3. Select **Bitwarden** &rarr; **Auto-fill** and then pick your login.
 
-Alternatively, you can also copy/paste your login credentials by accessing your vault in the popup from a regular, non-private browsing window.
+(2) Copy/paste your login credentials by accessing your vault in the popup from a regular, non-private browsing window.
```
Comment (fschillingeriv) [inline _articles/miscellaneous/extension-wont-load-in-private-mode.md:19]:
```suggestion
2. Copy/paste your login credentials by accessing your vault in the popup from a regular, non-private browsing window.
```

File: _articles/miscellaneous/extension-wont-load-in-private-mode.md [hunk 3]
```diff
@@ -9,13 +9,16 @@ tags: [firefox, extension, private window, private mode, private browsing]
...
+
+(3) Use the keyboard shortcut for auto-filling a login (CTRL + SHIFT + L by default on Windows, CMD + SHIFT + L by default on Mac)
```
Comment (fschillingeriv) [inline _articles/miscellaneous/extension-wont-load-in-private-mode.md:21]:
```suggestion
3. Use the keyboard shortcut for auto-filling a login (CTRL + SHIFT + L by default on Windows, CMD + SHIFT + L by default on Mac)
```

File: _articles/miscellaneous/extension-wont-load-in-private-mode.md [hunk 4]
```diff
@@ -9,13 +9,16 @@ tags: [firefox, extension, private window, private mode, private browsing]
...
 The Bitwarden browser extension does not completely function in Firefox's private browsing mode. This is a known issue specific only to Firefox. You will see a message indicating so when you try to open the Bitwarden popup window in a private window. We have discussed the problem with Mozilla, however, they seem unable to fix it so that extensions like Bitwarden can function entirely in private mode.
 
-As a workaround, if you are looking to auto-fill a login while in a private window you can use the right-click context menu:
+There are currently 3 workarounds for auto-filling a login while in Firefox's private browsing mode:
 
-1. Navigate to the website's login page while browsing in a private window.
-2. Right click the page. A context menu will appear.
-3. Select **Bitwarden** &rarr; **Auto-fill** and then pick your login.
+(1) Use the right-click context menu:
+  1. Navigate to the website's login page while browsing in a private window.
```
Comment (fschillingeriv) [inline _articles/miscellaneous/extension-wont-load-in-private-mode.md:15]:
```suggestion
   1. Navigate to the website's login page while browsing in a private window.
```

File: _articles/miscellaneous/extension-wont-load-in-private-mode.md [hunk 5]
```diff
@@ -9,13 +9,16 @@ tags: [firefox, extension, private window, private mode, private browsing]
...
+  2. Right click the page. A context menu will appear.
```
Comment (fschillingeriv) [inline _articles/miscellaneous/extension-wont-load-in-private-mode.md:16]:
```suggestion
   2. Right click the page. A context menu will appear.
```

File: _articles/miscellaneous/extension-wont-load-in-private-mode.md [hunk 6]
```diff
@@ -9,13 +9,16 @@ tags: [firefox, extension, private window, private mode, private browsing]
...
+  3. Select **Bitwarden** &rarr; **Auto-fill** and then pick your login.
```
Comment (fschillingeriv) [inline _articles/miscellaneous/extension-wont-load-in-private-mode.md:17]:
```suggestion
   3. Select **Bitwarden** &rarr; **Auto-fill** and then pick your login.
```

**Comment by fschillingeriv** [comment]
@MarcusOtter Thanks for the contribution, this is great! Made a few suggestions, only markdown-formatting.

**Comment by MarcusOtter** [comment]
👍 Thanks @fschillingeriv, was trying to get that to work for a while haha. Bit unsure about the wording of "[shortcut] **by default**" since you can't change it, but maybe that's good for futureproofing if you ever get the option to change the shortcuts? Or is it just misleading? Other than that I'm happy with it :)

**Review by fschillingeriv — APPROVED** [review]

**Pull request merged**